sdk/src/Services/S3/Custom/Transfer/_bcl+netstandard/TransferUtility.sync.cs

```diff
@@ -171,7 +171,7 @@ public void Upload(string filePath, string bucketName)
         {
             try
             {
-                UploadAsync(filePath, bucketName).Wait();
+                UploadWithResponseAsync(filePath, bucketName).Wait();
```

Comment:
actually i just realized this should say as UploadAsync because otherwise this would eventually create a span with `UploadAsyncWithResponse` instead of `UploadAsync`. will update that in next revision. same in other places below